config/initializers/training_routes.rb

```diff
@@ -91,7 +91,15 @@
   TRAINING_ROUTE_ENUMS.values_at(:assessment_only, :early_years_assessment_only).exclude?(training_route)
 }.freeze
 
-LEAD_PARTNER_ROUTES = %i[school_direct_salaried school_direct_tuition_fee pg_teaching_apprenticeship provider_led_postgrad provider_led_undergrad early_years_salaried iqts].freeze
+LEAD_PARTNER_ROUTES = %i[
+  school_direct_salaried
+  school_direct_tuition_fee
+  pg_teaching_apprenticeship
+  provider_led_postgrad provider_led_undergrad
```

Comment:
Split this line as well for readability?
```suggestion
  provider_led_postgrad
  provider_led_undergrad
```